recipe/build-mpi.sh

```diff
@@ -61,6 +63,7 @@ fi
             --enable-mpi-fortran \
             --disable-wrapper-rpath \
             --disable-wrapper-runpath \
+            --with-mpi-moduledir="$PREFIX/include" \
```

Comment:
done, though we don't do this for any other path, so now the installation is inconsistent with respect to path handling. The practical result ought to be the same.

Reply:
I do not understand. What's exactly is your concern? Doing it via `${includedir}` is better, because this way you do not break support for ompi's handling of the `OPAL_PREFIX` environment variable. If we use `${includedir}` for modules and later at runtime set `OPAL_PREFIX` to a different prefix, ompi will be able to properly relocate paths. That would not happen if you use `$PREFIX/include`. IIRC, we must override `OPAL_PREFIX` for cross-compilation.

Reply:
Oh, you are talking about these `--with-with-wrapper-*flags="$PREFIX/include"`. OK, to be honest, I do not remember why these are needed. Is this required to support cross-compilation?

Reply:
I don't know! I'm fine with this either way. Reading and thinking about the doc you linked, we should probably be using `${includedir}` / `${libdir}` systematically rather than $PREFIX in these args.

Reply:
Well, I'm not sure the `--with-with-wrapper-*flags` configure options actually support the `${includedir}` tag. 

Let me run test the thing locally and I'll  be back to you, maybe we can add the fix to this PR.

Reply:
OK, I tested locally, and using  `--with-with-wrapper-*flags=-I${includedir}'` works fine, and respects the OPAL_PREFIX feature.

HOWEVER, I do not see the need for passing these additional flags, it looks redundant. You  added them years ago in 32c607e0e58eb099fcdadb94696e1c25084f83a4, but many things have changed in conda-forge since then.
UNLESS... we somehow need these flags for cross-compilation to work, but I doubt it, because we can use (and we are using) the OPAL_PREFIX mechanism if needed.

IMHO, I would try removing all these `--with-with-wrapper-*flags` from configure and ross fingers. What do you thing, @minrk?

Reply:
Oh, but maybe `--disable-wrapper-rpath` and `--disable-wrapper-runpath` should also be removed? Because it looks like you did want to pass `-Wl,-rpath` flags to the linker.

Reply:
I don't know why `wrapper-rpath` was replaced with `wrapper-ldflags`. I'm hesitant to change it, but it was done long ago in #35 and I don't recall why.